src/components/Header/Menu/Button/index.js

```diff
@@ -0,0 +1,13 @@
+import "./style.css";
+
+function MenuButton(props) {
```

Comment:
Start to write and use arrow functions everywhere

```suggestion
const MenuButton = (props) => {
```